src/Peachpie.Library/Resources/Resources.resx

```diff
@@ -681,4 +681,7 @@
   <data name="pcre_make_error" xml:space="preserve">
     <value>{0} at offset {1}</value>
   </data>
+  <data name="ftp_error_no_nb" xml:space="preserve">
+    <value>No nbronous transfer to continue.</value>
```

Comment:
Put _asynchronous_ there - unlike PHP we are trully asynchronous and _nbronous_ is a really weird word :-)

Reply:
@jakubmisek If it does, I wasn't able to find it. The docs of the `ftp_nb_*` functions always mention only _asynchronous_ or _non-blocking_, so this word doesn't make any sense. If we want to keep closer to the original, I'd suggest: "No asynchronous (non-blocking) transfer to continue."